.github/workflows/ci.yml

```diff
@@ -0,0 +1,27 @@
+name: CI
+
+on: [push, pull_request]
+
+jobs:
+  codespell:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: codespell-project/actions-codespell@v2
+        with:
+            ignore_words_file: spelling.ignore.txt
+            skip: alectryon
+  build:
+    strategy:
+      fail-fast: false
+      matrix:
+        project: ["c", "haskell"] # coq takes a very long time and times out at 6 hours...
+        os: ["ubuntu-latest", macos-latest]
```

Comment:
Are there `ubuntu-stable` and `macos-stable` that we can run as well?

Reply:
https://github.com/actions/runner-images

Reply:
Ah maybe I misunderstand.  I'm looking to run both nixpkgs stable and nixpkgs weekly.

Reply:
Oh I see, sure I’ll look into that